src/chart/pie/PieView.ts

```diff
@@ -185,6 +185,10 @@ class PiePiece extends graphic.Sector {
             z2: 10
         });
 
+        if (seriesModel.get('label').position !== 'outside') {
```

Comment:
Should always use `seriesModel.get(['label', 'position'])` to avoid null value access.
`outer` should also be considered.

```ts
const position = seriesModel.get(['label', 'position']);
if (position !== 'outside' && position !== 'outer')
```